components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1059,8 +1098,11 @@ public AccessTokenDO getAccessToken(String accessTokenIdentifier, boolean includ
                     String subjectIdentifier = resultSet.getString(14);
                     String authenticatedIDP = null;
                     String tokenBindingReference = resultSet.getString(15);
+                    String authorizedOrganization = resultSet.getString(16);
+                    int appResideTenantId = resultSet.getInt(17);
+
                     if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
-                        authenticatedIDP = resultSet.getString(16);
+                        authenticatedIDP = resultSet.getString(18);
```

Comment:
Relates to this comment - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2187/files#r1359867338
The number is increased by 2 as two params were introduced.